pages/overview/what-is-jsonschema.md

```diff
@@ -2,219 +2,51 @@
 section: docs
 title: What is JSON Schema?
 ---
-JSON Schema is a declarative language that you can use to annotate and validate the structure, constraints, and data types of your JSON documents. It provides a way to standardize and define expectations for your JSON data.
+JSON Schema is a declarative language for annotating and validating JSON documents' structure, constraints, and data types. It provides a way to standardize and define expectations for JSON data.
 <br/><br/>
 ![How JSON Schema works](/img/what-is-json-schema.png)
```

Comment:
The image is missing in the PR. Independently of the content of the image I think it will require a section called "How JSON Schema works" and an intro or explanatory paragraph before or after the image.

Reply:
The image is showing on the dev server @benjagm 
![Screenshot (531)](https://github.com/json-schema-org/website/assets/110067624/dd482ae1-fcc7-4309-8753-8a9cf7e2ca66)

Reply:
This is image is just an illustration art-work that does not explain how JSON Schema works and I think we should not use it here.

I think we'll need something else that clearly explain how JSON Schema works. What about something like this (this is a raw draft for discussion) with notes and clarifications.

<img width="979" alt="Screenshot 2024-05-13 at 12 25 29" src="https://github.com/json-schema-org/website/assets/40007659/b4fb8054-5834-48de-a26f-433297f4c69f">

More information and editable presentation [here](https://docs.google.com/presentation/d/1WYhedTV64JK00YWj0o-fOQN0FNrcZRfOrcZaJnKZqM8/edit?usp=sharing) and more ideas.

I like how @jviotti presented diagrams in this blog post and I think we should do something similar.
https://json-schema.org/blog/posts/understanding-lexical-dynamic-scopes